test/test_torch.py

```diff
@@ -7804,6 +7804,38 @@ def test_view(self, device):
         self.assertEqual(tensor.view(6, 2, 1), contig_tensor.view(6, 2, 1))
         self.assertEqual(tensor.view(1, 6, 2, 1), contig_tensor.view(1, 6, 2, 1))
 
+    @onlyOnCPUAndCUDA
+    def test_view_dtype(self, device):
```

Comment:
This is cool but how about pairing dtypes, like original and converted, so you have (f32, i32), (i16, f16), (double, complex64), for example? You can test against NumPy and test for cyclic correctness to verify their values. By "cyclic correctness" I mean that this can test that double -> complex64 -> double gives the same value.

Reply:
fixed